### ISSUE TITLE
Multiple cloud resources for lessons

### ISSUE THREAD

**Issue by tracykteal** [datacarpentry/cloud-genomics#7]
There are multiple possibilities for cloud resources for these workshops. Places hosting multiple workshops that have local resources or organizations that have resources might want to use their own. We therefore likely need parallel documentation on those resources for components of these lessons. 

Likely we should keep them all in this repo and name them 01-aws-logging-in-to-cloud.md and 01-iplant-logging-in-to-cloud.md but maybe we need to think about how to manage this and what cloud resources we want to develop lessons for.

**Comment by devbioinfoguy** [comment]
Yes, I noticed that there's a lot in one lesson (login) that is specific to the cloud provider. Will this be moved out to Instructor's notes? Or do we think that individuals will want to launch their own instances under iPlant (Cyverse) or some other provider for their own use / learning?

**Comment by JasonJWilliamsNY** [comment]
I like @tracykteal idea to just have different versions!

**Comment by devbioinfoguy** [comment]
Absolutely. I'm in agreement with that!  :-)
